src/itemTracker/MapDungeonTracker.js

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+import Col from 'react-bootstrap/Col';
+// import { Stack } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Item from './Item';
+
+import noSmallKey from '../assets/dungeons/noSmallKey.png';
+import oneSmallKey from '../assets/dungeons/1_smallKey.png';
+import twoSmallKey from '../assets/dungeons/2_smallKey.png';
+import threeSmallKey from '../assets/dungeons/3_smallKey.png';
+import noEntrance from '../assets/No_Entrance.png';
+import entrance from '../assets/Entrance.png';
+import DungeonName from './items/dungeons/DungeonName';
+import ColorScheme from '../customization/ColorScheme';
+import Logic from '../logic/Logic';
+
+class MapDungeonTracker extends React.Component {
```

Comment:
This component duplicates a lot of stuff from `DungeonTracker` - it might be useful to just pass a `mapMode` prop to `DungeonTracker`, which causes it to render the reduced dungeon tracker.

Reply:
It'll likely be better long term to try and componetize `DungeonTracker` further. Accomplishes the same goal of reducing the code duplication., without necessarily introducing additional maintenance overhead with multiple codepaths and conditional rendering, especially if/when the layout options are further expanded